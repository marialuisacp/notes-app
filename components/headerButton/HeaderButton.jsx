import React from 'react';
import StyledHeaderButton from './styles/HeaderButton.styles';
import StyledHeaderButtonText from './styles/HeaderButtonText.styles';

const HeaderButton = ({ icon }) => (
  <StyledHeaderButton>
    <StyledHeaderButtonText className={`icon-${icon}`}></StyledHeaderButtonText>
  </StyledHeaderButton>
);

export default HeaderButton;