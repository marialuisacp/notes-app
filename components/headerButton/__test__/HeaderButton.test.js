import React from 'react';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import HeaderButton from '../HeaderButton';

test('should render component HeaderButton', () => {
  const { HeaderButtonFragment } = render(<HeaderButton></HeaderButton>);
  expect(HeaderButtonFragment).toMatchSnapshot();
})
