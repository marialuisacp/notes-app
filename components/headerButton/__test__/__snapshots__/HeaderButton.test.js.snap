// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component HeaderButton 1`] = `undefined`;
